static/src/javascripts/projects/commercial/modules/header-bidding/utils.ts

```diff
@@ -141,14 +139,11 @@ export const shouldIncludeAppNexus = (): boolean =>
 export const shouldIncludeXaxis = (): boolean => isInUk();
 
 export const shouldIncludeImproveDigital = (): boolean => isInUk() || isInRow();
-export const shouldIncludeImproveDigitalSkin = (): boolean => {
-	return (
-		isInVariantSynchronous(improveSkins, 'variant') &&
-		window.guardian.config.page.isFront &&
-		isInUk() &&
-		getBreakpointKey() === 'D' // Desktop only
-	);
-};
+export const shouldIncludeImproveDigitalSkin = (): boolean =>
+	!!window.guardian.config.switches.prebidImproveDigitalSkins &&
+	window.guardian.config.page.isFront &&
+	isInUk() &&
```

Comment:
Let me just double check this with Spencer. I think collective might be RoW as well.

Reply:
There’s [a placement ID for RoW, indeed][row], so it probably should be launched there, too.

[row]: https://github.com/guardian/frontend/blob/e7c4d1798d347f3b5e205982ca8d5162bc36f96c/static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.ts#L178